src/app/components/HabitPlant/HabitPlant.stories.tsx

```diff
@@ -6,10 +6,16 @@ export default {
   component: HabitPlant,
 };
 
-export const NewlyPlanted = (): JSX.Element => <HabitPlant rating={0} />;
+export const NewlyPlanted = (): JSX.Element => (
+  <HabitPlant rating="empty_pot" />
```

Comment:
`rating="empty_pot"`  has a different naming convention like you had before. 
What about `rating="emptyPot"`?
I would expect that a rating is a number or a string like "perfect", "beginner", "professional", but I guess it is similar to your approach.